### PR TITLE
Remove aircraft defaults

### PR DESCRIPTION
**Pull request by teees**
- Removed those pesky aircraft defaults for `RepairBuildings` and `RearmBuildings` that were only ever useful in ra
- Merged common ra aircraft definitons

### CODE REVIEW

File: mods/ra/rules/defaults.yaml [hunk 1]
```diff
@@ -388,6 +392,9 @@
 	Inherits: ^Plane
 	Tooltip:
 		GenericName: Helicopter
+	Aircraft:
+		RearmBuildings: hpad
```
Comment (cjshmyr) [inline mods/ra/rules/defaults.yaml]: Are we missing `RepairBuildings: fix` here?

Comment (teees) [inline mods/ra/rules/defaults.yaml]: That gets inherited from ^Plane:

Comment (cjshmyr) [inline mods/ra/rules/defaults.yaml]: Missed that, sounds good

**Comment by abcdefg30** [comment]
Looks good to me. :+1: / :white_check_mark:

**Comment by obrakmann** [comment]
:+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/9138307e93f1962912d0f7db18aef3582d3bae52)

**Pull request merged**